src/Tests/target_wrapper.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Was this file intended to be committed?

You can add these commands to your `.profile`, and then it'll be a part of your environment.
http://www.theunixschool.com/2011/07/what-is-profile-file.html?m=1
https://unix.stackexchange.com/questions/40708/what-is-the-difference-between-profile-bashrc-bash-profile-gnomer